### PR TITLE
Showing cancelled appointments on the resource calendar

### PR DESCRIPTION
**Pull request by Guntrisoft**
- cancelled appointments are now red (and transparent)
- moved appointments are now altered to be green

### CODE REVIEW

File: app/assets/javascripts/calendars/guiders-appointments.es6 [hunk 1]
```diff
@@ -118,12 +118,13 @@
         );
       }
 
+      element.removeClass('fc-event--moved');
+      element.removeClass('fc-event--cancelled');
```
Comment (benbarnett) [inline app/assets/javascripts/calendars/guiders-appointments.es6]: You can space separate these in one call

**Comment by benlovell** [comment]
Can you rebase this and try again?

File: app/assets/javascripts/calendars/guiders-appointments.es6 (hunk 1 above)
Comment (Guntrisoft) [inline app/assets/javascripts/calendars/guiders-appointments.es6]: Done

**Comment by Guntrisoft** [comment]
@benbarnett good to go?

**Comment by benbarnett** [comment]
Hit it!

**Pull request merged**